tasks.py

```diff
@@ -0,0 +1,86 @@
+import os
+import platform
+from datetime import datetime
+from pathlib import Path
+
+from robocorp import windows
+from robocorp.tasks import setup, task
+from robocorp.windows import WindowElement
+
+
+desktop = windows.Desktop()
+window = None  # Notepad window object after being controlled
+
+LOCATOR_NOTEPAD = 'class:"Notepad" subname:"Notepad"'
+OPEN_FILE = str(os.getenv("FILE_TO_OPEN", Path("devdata") / "test.txt").absolute())
+
+
+def get_win_version() -> str:
+    """Windows only utility which returns the current Windows major version."""
+    # Windows terminal `ver` command is bugged, until that's fixed, check by build
+    #  number. (the same applies for `platform.version()`)
+    version_parts = platform.version().split(".")
+    major = version_parts[0]
+    if major == "10" and int(version_parts[2]) >= 22000:
+        major = "11"
+
+    return major
+
+
+@setup
+def open_notepad(task):
+    """Opens Notepad for automation, then closes it at the end."""
+    desktop.windows_run(f"notepad.exe {OPEN_FILE}")
+    yield
+    desktop.close_windows(LOCATOR_NOTEPAD)
+
+
+def _set_property(unique_id: str, value: str):
+    field = window.find(f'id:"{unique_id}"')
+    field.click()
+    field.send_keys("{DOWN}", wait_time=0.5)
+    desktop.send_keys(value, send_enter=True, wait_time=0.5)
+
+
+def _select_property(unique_id: str, value: str):
+    field = window.find(f'id:"{unique_id}"')
+    field.ui_automation_control.Select(value)
```

Comment:
In general the `ui_automation_control` field should not be accessed.

This method could be removed as the following could be written directly:  

`window.find(f'id:"{unique_id}"').select(value)`

or alternatively:

`window.select(value, locator=f'id:"{unique_id}"')`

Reply:
Agree with this, hide the "internals" as much as possible as these examples are the ones that ReMark will suggest and users will imitate.

Reply:
And ReMark does a great job at it, maybe the first time I tried it it didn't lead me to the answer I'm getting now:

<img width="911" alt="Screenshot 2024-01-23 at 1 42 41 PM" src="https://github.com/robocorp/example-windows-notepad/assets/709053/4f0d0dcf-433a-46ed-8d25-eb0a1cdc4a60">